Library/Homebrew/cmd/install.rb

```diff
@@ -100,7 +100,8 @@ def install
         query = query_regexp(e.name)
 
         ohai "Searching for similarly named formulae..."
-        formulae_search_results = search_formulae(query)
+        formulae_search_results = search_formulae(query, { :installed => false })
```

Comment:
I think its better and more customary to drop the braces and rely on Ruby to collect those “named arguments” into a hash and pass that as the last argument.